Sources/JoyfillComponents/Components/Chart/Chart.swift

```diff
@@ -189,9 +191,11 @@ public class Chart: UIView, UIViewControllerTransitioningDelegate {
         horizontalLabel.textAlignment = .center
         horizontalLabel.labelText = "Horizontal"
         
-        if xCoordinates[index].isEmpty != true {
-            for i in 0...yCoordinates[index].count - 1 {
-                lineGraph.addLine(yCoordinates[index][i], labels: graphLabelData[index][i])
+        if index <= xCoordinates.count {
```

Comment:
Please comment through this about what i going on here and what this does

Reply:
This is used to create graph coordinates only if xCoordinates array contains value greater than or equal to chart indexPath. This is done to prevent crashing while passing coordinates to the graph.